File: Frontend/src/app/Shared/VendorOnboardRequestVM.ts
import {VendorOnboardRequest} from "./VendorOnboardRequest";

export interface VendorOnboardRequestVM {
    onboard_Request_Id : number;
    vendor_ID: number;
    onboard_Request_status_ID:number;
    employeeName: string;
    vendors: VendorOnboardRequest;
    quotes: string;
}

File: Backend/ViewModel/VendorOnboardRequestVM.cs
using ProcionAPI.Models.Entities;

namespace ProcionAPI.ViewModel
{
    public class VendorOnboardRequestVM
    {
        public int Onboard_Request_Id { get; set; }
        public int Vendor_ID { get; set; }
        public int Onboard_Request_status_ID { get; set; }
        public string EmployeeName { get; set; }
        public Vendor Vendors { get; set; }
        public string Quotes { get; set; }
    }
}
